portfolio/src/main/webapp/index.html

```diff
@@ -40,6 +40,22 @@ <h2>Links</h2>
                 <li><a href="https://github.com/abiirami">Github</a></li>
             </ul>
         </div>
+        <form action="/data" method="POST">
+            <p>Comments?</p>
+            <textarea name="text-input">Enter any comments here...</textarea>
```

Comment:
Nit: change text-input --> comments-input just to make it absolutely clear what param you're getting in DataServlet.java. This will also help in case you decide to add more text inputs (e.g. username? email address?)